codebase/superdarn/src.lib/tk/iq.1.7/src/iqwrite.c

```diff
@@ -61,7 +61,7 @@ int IQEncode(struct DataMap *ptr,struct IQ *iq,
   tnum=iq->seqnum;
   bnum=iq->tbadtr*2;
 
- snum=iq->seqnum*iq->chnnum*iq->smpnum*2*2;   /*2 for main and back  2  to convert to 16 bit */  
+  snum=iq->seqnum*iq->chnnum*iq->smpnum*2;
```

Comment:
Recommend adding a comment above that describes what's being done that is different than before.

Reply:
@egthomas, is this something that you worked on and would be able to provide a comment here?